findbugs.properties

```diff
@@ -1,11 +1,17 @@
 category=External Analysers
 description=Provide Findbugs rules for analysis of Java projects
-archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0
-publicVersions=3.9.4,3.11.0
+archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0
+publicVersions=3.9.4,3.11.1
 
 defaults.mavenGroupId=org.sonarsource.sonar-findbugs-plugin
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+3.11.1.description=Use SpotBugs 3.1.12
```

Comment:
Did you copy/paste and forget to update the description?

Reply:
no the version of spotbugs hasn't been changed.

Reply:
Your description here does not match the description on the release. As a user if I look at your current description I will be far less motivated to upgrade than if I see "Fix CVE-2019-10173". _That_ should be the description for this version.

Reply:
updated.